packages/special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -808,12 +809,18 @@
 
     const page = new DuckPlayerPageMessages(messaging, import.meta.injectName)
 
-    await Comms.init(page)
+    const result = await Comms.init(page)
 
-    if (!Comms.messaging) {
-        console.warn('cannot continue as messaging was not resolved')
+    if (!('value' in result)) {
+        console.warn('cannot continue as the initialSetup call didnt complete');
+        console.error(result.error);
         return
     }
+
+    // here we have access to userValues and settings
+    console.log(result.value.userValues)
+    console.log(result.value.settings)
```

Comment:
@Bunn - just a small example showing how to access the new values - you can then feed them into individual features as needed

Reply:
Awesome, LGTM
https://github.com/duckduckgo/macos-browser/pull/2830#issuecomment-2145037640